VirtoCommerce.JavaScriptShoppingCart.Web/Scripts/checkout/checkout-address.js

```diff
@@ -58,14 +58,15 @@ cartModule.component('vcCheckoutAddress', {
 				ctrl.address.regionName = undefined;
 			}
 		}
+		ctrl.setForm = function (frm) { ctrl.form = frm; };
 
 		ctrl.validate = function () {
```

Comment:
Is this ok that func Validate returns a value? Usually, validate function shouldn't return any value, just void

Reply:
I agree with you. Possible is a better way to call it IsValid. But here a lot of changes in different checkout-* blades. https://github.com/VirtoCommerce/vc-module-javascript-shoppingcart/blob/a883fa31be61d483613386f84150f2e8958ff97a/VirtoCommerce.JavaScriptShoppingCart.Web/Scripts/checkout/checkout-wizard-step.js#L32. 
It's out of the scope of this fix.